test/reply-code.test.js

```diff
@@ -57,3 +57,37 @@ test('code should handle null/undefined/float', t => {
     t.equal(res.statusCode, 404)
   })
 })
+
+test('code should handle 204', t => {
```

Comment:
Out of @climba03003 comment, we should add a test that verifies that if the `onSend` tries to alter the body, this alteration is ignored.